pages/docs/reason-compiler/latest/pipe-last.mdx

```diff
@@ -0,0 +1,22 @@
+# Pipe-last
```

Comment:
Looks like a duplication of the doc page in the Reason manual. Since this section covers the BuckleScript compiler, I think we don't need it here.

Reply:
@yawaramin

I found where `pipe-first` is documented in this repo and added the corresponding changes for `pipe-last`.

I saw `pipe-first` in two places too. I don't know which one should be deleted. Are you suggesting we should delete `pipe-first` and `pipe-last` in Reason manual and we keep them in the BS compiler section?

Reply:
To be honest I think covering the pipe-first operator in such detail in the [Reason Manual](https://github.com/reason-association/reasonml.org/blob/5cb14def49625860d854498f8a4ec02eedfe6109/pages/docs/manual/latest/pipe-first.mdx) is redundant, because it's already covered in the [BuckleScript Manual](https://github.com/reason-association/reasonml.org/blob/5cb14def49625860d854498f8a4ec02eedfe6109/pages/docs/reason-compiler/latest/pipe-first.mdx) and in any case it can't be used in native Reason. Anyway, that's out of scope for this PR.

Here I would still recommend to not cover pipe-last in the BuckleScript Manual because it's not really a BuckleScript-specific thing, we just want to make sure we at least cover it in the Reason Manual since it's common to all target platforms.

Reply:
@yawaranmin Added a commit to remove pipe-last in the BuckleScript Manual. Can squash if @ryyppy approves it. I leave removing duplications of pipe-first doc to later when the plan to dedupe is more concrete.